.github/workflows/lslint.yml

```diff
@@ -0,0 +1,25 @@
+name: LSLint
+
+on: [push, pull_request,workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      LSLINT_VER: v1.2.6
```

Comment:
This could be made more efficient if we were to run the makefile to compile LSLint. No version number would be needed.  GHA is designed to compile software, Therefore it has a entire toolchain for compiling C and C++.

Reply:
That is true, but please note that LSLINT has not received any updates since August 2020, so building lslint every time the workflow runs is a bit exessive imo.